packages/xo-server/src/xo-mixins/rest-api.mjs

```diff
@@ -208,6 +208,28 @@ async function _getDashboardStats(app) {
     dashboard.missingPatches = missingPatches
   }
 
+  try {
+    const backupRepositorySize = Object.values(await app.getAllRemotesInfo()).reduce(
+      (prev, remoteInfo) => ({
+        available: prev.available + remoteInfo.available,
+        backups: 0, // @TODO: compute the space used by backups
+        other: 0, // @TODO: compute the space used by everything that is not a backup
+        total: prev.total + remoteInfo.size,
+        used: prev.used + remoteInfo.used,
+      }),
+      {
+        available: 0,
+        backups: 0,
+        other: 0,
+        total: 0,
+        used: 0,
+      }
+    )
+    dashboard.backupRepository = { size: backupRepositorySize }
```

Comment:
There is no single backup repository, IMHO it should be plural otherwise it's confusing.